osu.Game/Screens/Select/BeatmapInfoWedge.cs

```diff
@@ -69,6 +69,7 @@ public void UpdateBeatmap(WorkingBeatmap beatmap)
                 beatmapInfoContainer?.FadeOut(250);
                 beatmapInfoContainer?.Expire();
                 beatmapInfoContainer = null;
+                FadeOut(250);

```

Comment:
This doesn't fade out the whole left area, just gets rid of the transparent blue background from the wedge in the upper left corner. Without this the image disappears and it tilts slightly down but doesn't go away.